tests/functional/heketi/test_heketi_brick_evict.py

```diff
@@ -110,10 +110,29 @@ def _wait_for_gluster_pod_after_node_reboot(self, node_hostname):
             openshift_ops.check_service_status_on_pod(
                 self.ocp_client, gluster_pod, service, "active", state)
 
+    def _power_off_node_and_wait_node_to_be_not_ready(self, hostname):
+        # Bring down the glusterfs node
+        vm_name = node_ops.find_vm_name_by_ip_or_hostname(hostname)
+        self.addCleanup(
+            self._wait_for_gluster_pod_after_node_reboot, hostname)
+        self.addCleanup(node_ops.power_on_vm_by_name, vm_name)
+        node_ops.power_off_vm_by_name(vm_name)
+
+        # Wait glusterfs node to become NotReady
+        custom = r'":.status.conditions[?(@.type==\"Ready\")]".status'
+        for w in waiter.Waiter(300, 20):
+            status = openshift_ops.oc_get_custom_resource(
+                self.ocp_client, 'node', custom, hostname)
+            if status[0] in ['False', 'Unknown']:
+                break
+        if w.expired:
+            raise exceptions.ExecutionError(
+                "Failed to bring down node {}".format(hostname))
+
     @pytest.mark.tier4
-    def test_brick_evict_with_node_down(self):
+    def test_brick_evict_with_node_down_with_three_node(self):
         """Test brick evict basic functionality and verify brick evict
-        after node down"""
+        will fail after node down if nodes are three"""
```

Comment:
Test brick evict basic functionality, on three nodes with one node down

Reply:
I have added more descriptions which are giving more information about the function. Your suggestion does not give more information about the function.